src/views/Projects/index.jsx

```diff
@@ -0,0 +1,86 @@
+import { Component, Fragment } from 'react';
+import { withStyles } from 'material-ui/styles';
+import Grid from 'material-ui/Grid';
+import Button from 'material-ui/Button';
+import Typography from 'material-ui/Typography';
+import projects from '../../data/loader';
+import ProjectCard from '../../components/ProjectCard';
+
+@withStyles(
+  theme => ({
+    grid: {
+      padding: '0 15px !important',
+      maxWidth: '100%',
+    },
+    content: {
+      flexGrow: 1,
+      backgroundColor: theme.palette.background,
+      minWidth: 0,
+    },
+    container: {
+      paddingRight: '15px',
+      paddingLeft: '15px',
+      marginRight: 'auto',
+      marginLeft: 'auto',
+    },
+    flex1: {
+      flex: 1,
+    },
+    columnFlex: {
+      display: 'flex',
+      flexDirection: 'column',
+    },
+  }),
+  { withTheme: true }
```

Comment:
This is only needed when you use `theme` inside the class. I don't think we are using it so we can remove it.